simplify

efjacobson opened the pull request:
@gravelfreeman unless i'm missing something, i believe this does the same as your script while solving your depth problem. from `man cp`

```
       -l, --link
              hard link files instead of copying

       -R, -r, --recursive
              copy directories recursively
```

efjacobson committed:
simplify

gravelfreeman commented:
Hello! Have you tried it and confirm this is working? Reason is that I found a script that does what I needed but better imo. I'm still leaving this one on my Github since some people are using it. If you've tested it and you're confirming that it's working I'll go on with the pull request.

Thank you!

efjacobson commented:
@gravelfreeman i haven't used your script as i have my own but i use `cp -lR` all the time and it does what it says on the box (it hardlinks a file or entire directory). no need to merge the PR i just thought you might like to know about this method.

what is the new and improved script that you are using these days?

gravelfreeman commented:
I'm using Paul Chamber's [script](https://gist.github.com/paul-chambers/71ef48e40449ec73eef95430b9e4e6c7).

If anyone test the change here and confirms that it's working, I'll merge.

Thanks!

efjacobson commented:
@gravelfreeman yep - that script uses the [same method](https://gist.github.com/paul-chambers/71ef48e40449ec73eef95430b9e4e6c7#file-torrent-complete-sh-L56)

gravelfreeman commented:
Sorry when I made this repo I didn't knew how to code and use Github. I since made some progress on that front. I updated and cleaned the script. I think you'll like it now, it does everything one would want and the code is super light.

efjacobson commented:
> Sorry when I made this repo I didn't knew how to code and use Github. I since made some progress on that front. I updated and cleaned the script. I think you'll like it now, it does everything one would want and the code is super light.

nice! no need to apologize, this is your repo after all :] i just happened to stumble upon it

no idea where i heard it but (the polite version of) a quote that has stuck with me goes something like "when you look back at code you wrote in the past and see all the flaws, that's a good thing - it means you are getting better"
